creator-node/test/lib/utils.js

```diff
@@ -0,0 +1,7 @@
+const sortKeys = x => {
```

Comment:
lets make sure this function is shared by both the test and the code. we don't want to be changing the underlying code but not the test

Reply:
could you explain more about this? I added this fn to check that metadata contents match what is expected, not so much for actual logic

Reply:
i think the function is now defined exactly the same in two places,  apiHelpers and test/lib/utils.js. what's the difference between the two?

Reply:
ah there is no difference. I thought itd be a good idea to have a libs util file, but I can very easily instead use this sort method from `apiHelpers.js`. will refactor to use method from `apiHelpers.js`.